integration/test/ca_bundle/ca_bundle_test.go

```diff
@@ -0,0 +1,117 @@
+package ca_bundle
```

Comment:
We should probably add
```
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: MIT

// +build linux
```
at the top of this go file.
Based on all the paths and the calls to `exec.Command()` I'd say it won't run on a windows host.